elasticai_api/common/data_shard_service.py

```diff
@@ -125,12 +130,45 @@ def report_batch_done(self, batch_size=None, err_msg=""):
             return True
         return False
 
-    def fetch_shard(self):
```

Comment:
TF 1.x custom training loop relies on this method. What's the consideration of removing it?

Reply:
Yes, it is a mistake and I have restored it.